db/seeds.rb

```diff
@@ -32,7 +32,8 @@
   reason_to_donate: 'I support inclusive education in tech',
   slug: 'railsbridge',
   info_url: 'http://www.railsbridge.com',
-  donation_url: 'http://www.railsbridge.org/help/donate'
+  donation_url: 'http://www.railsbridge.org/help/donate',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.